src/basic_recipes/title.jl

```diff
@@ -0,0 +1,30 @@
+@recipe(Title, titletext, plot) do scene
+    Theme(
+        textsize=40,
+        raw=true,
```

Comment:
What happens if `raw = false`?  As I understand it, since it's just text, `raw` should always be true ;)

Reply:
(by which I mean this doesn't seem necessary as an Attribute)

Reply:
If I do not explicitly pass `raw=true` in the `text!` call I get 
```julia
MethodError: no method matching isfinite(::Nothing)
Closest candidates are:
  isfinite(!Matched::Float16) at float.jl:548
  isfinite(!Matched::Missing) at missing.jl:83
  isfinite(!Matched::BigFloat) at mpfr.jl:882
  ...

Stacktrace:
 [1] data_limits(::Array{AbstractPlot,1}) at /home/jan/.julia/dev/AbstractPlotting/src/layouting/data_limits.jl:113
 [2] data_limits at /home/jan/.julia/dev/AbstractPlotting/src/layouting/data_limits.jl:119 [inlined]
 [3] update_limits!(::Scene, ::AbstractPlotting.Automatic, ::Vec{3,Float32}) at /home/jan/.julia/dev/AbstractPlotting/src/interfaces.jl:664
 [4] update_limits!(::Scene) at /home/jan/.julia/dev/AbstractPlotting/src/interfaces.jl:660
 [5] plot!(::Scene, ::Type{Title{...}}, ::Attributes, ::Tuple{Observables.Observable{String},Observables.Observable{Scene}}, ::Observables.Observable{Tuple{String,Scene}}) at /home/jan/.julia/dev/AbstractPlotting/src/interfaces.jl:557
 [6] #plot!#195(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Scene, ::Type{Title{...}}, ::Attributes, ::String, ::Vararg{Any,N} where N) at /home/jan/.julia/dev/AbstractPlotting/src/interfaces.jl:514
 [7] plot! at /home/jan/.julia/dev/AbstractPlotting/src/interfaces.jl:483 [inlined]
 [8] #title#620 at /home/jan/.julia/dev/AbstractPlotting/src/recipes.jl:17 [inlined]
 [9] (::getfield(AbstractPlotting, Symbol("#kw##title")))(::NamedTuple{(:color, :strokecolor),Tuple{Symbol,Symbol}}, ::typeof(title), ::String, ::Scene) at ./none:0
 [10] top-level scope at In[3]:6
```

Reply:
Right, but you can do raw=true in text without it being a keyword argument.